pyArango/document.py

```diff
@@ -373,21 +386,49 @@ def getPatches(self) :
         """return the patches in a dict format"""
         return self._store.getPatches()
 
+    def __dir__(self):
+        if not self._store:
+            return []
+        return dir(self._store)
+        
+    def __len__(self):
+        if not self._store:
+            return 0
+
+        return self._store.__len__()
+
+    def __dict__(self):
+        if not self._store:
+            return []
+        return dict(self._store)
```

Comment:
The function should return a dictionary.
```python
    def __dict__(self):
        if not self._store:
            return {}
        return dict(self._store)
```

Reply:
thanks, fixed.